README.md

```diff
@@ -1,3 +1,108 @@
-# imgix-api-js
+<!-- ix-docs-ignore -->
 
-A JS library that wraps the imgix public API
+![imgix logo](https://assets.imgix.net/sdk-imgix-logo.svg)
+
+`imgix-management-js` is a JavaScript client library for making requests against the [imgix](https://www.imgix.com/) Management API.
```

Comment:
Do you think we should mention that this is "experimental" and likely to be merged into imgix-core-js in the future?